src/driver/rtc.h

```diff
@@ -0,0 +1,18 @@
+#ifndef _RTC_H
+#define _RTC_H
+
+struct rtc_date {
```

Comment:
we are using the `typedef struct ...` pattern elsewhere. any reason we can not employ it here?

Reply:
We could but I was keeping the implementation similar to rtc_time for consistency.  If you prefer typedef I can change it.

Reply:
ok, yeah, i can see that. fair enough.